presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.h

```diff
@@ -25,7 +25,7 @@ class PartitionAndSerializeNode : public velox::core::PlanNode {
   PartitionAndSerializeNode(
       const velox::core::PlanNodeId& id,
       std::vector<velox::core::TypedExprPtr> keys,
-      int numPartitions,
+      int32_t numPartitions,
```

Comment:
Can we have it as uint32_t? I assume this wouldn't be negative?

Reply:
yeah. updated.